hazelcast-documentation/src/HazelcastSimulator/ExecutingASimulatorTest.md

```diff
@@ -4,17 +4,311 @@
+INFO  13:25:41 Artifact: /disk1/hasan/.m2/repository/com/hazelcast/hazelcast-wm/3.5/hazelcast-wm-3.5.jar is not found in local maven repository, trying online one
+INFO  13:25:50 Created compute
+INFO  13:25:50 Machine spec: hardwareId=m3.medium,locationId=us-east-1,imageId=us-east-1/ami-fb8e9292
+INFO  13:25:56 Created template
+INFO  13:25:56 Login name to the remote machines: simulator
+INFO  13:25:57 Security group: 'simulator' is found in region 'us-east-1'
+INFO  13:25:57 Creating machines... (can take a few minutes)
+INFO  13:27:10     54.157.207.79 LAUNCHED
+INFO  13:27:10     54.205.76.229 LAUNCHED
+INFO  13:27:10     54.158.159.32 LAUNCHED
+INFO  13:27:10     54.157.2.215 LAUNCHED
+INFO  13:27:21     54.157.207.79 JAVA INSTALLED
+INFO  13:27:24     54.205.76.229 JAVA INSTALLED
+INFO  13:27:27     54.158.159.32 JAVA INSTALLED
+INFO  13:27:28     54.157.2.215 JAVA INSTALLED
+INFO  13:27:33     54.157.207.79 SIMULATOR AGENT INSTALLED
+INFO  13:27:33 Killing Agent on: 54.157.207.79
+INFO  13:27:33 Starting Agent on: 54.157.207.79
+INFO  13:27:33     54.157.207.79 SIMULATOR AGENT STARTED
+INFO  13:27:35     54.205.76.229 SIMULATOR AGENT INSTALLED
+INFO  13:27:35 Killing Agent on: 54.205.76.229
+INFO  13:27:35 Starting Agent on: 54.205.76.229
+INFO  13:27:35     54.205.76.229 SIMULATOR AGENT STARTED
+INFO  13:27:40     54.158.159.32 SIMULATOR AGENT INSTALLED
+INFO  13:27:40 Killing Agent on: 54.158.159.32
+INFO  13:27:40 Starting Agent on: 54.158.159.32
+INFO  13:27:40     54.158.159.32 SIMULATOR AGENT STARTED
+INFO  13:27:46     54.157.2.215 SIMULATOR AGENT INSTALLED
+INFO  13:27:46 Killing Agent on: 54.157.2.215
+INFO  13:27:50 Starting Agent on: 54.157.2.215
+INFO  13:27:51     54.157.2.215 SIMULATOR AGENT STARTED
+INFO  13:27:51 Pausing for machine warmup... (10 sec)
+INFO  13:28:01 Duration: 00d 00h 02m 24s
+INFO  13:28:01 ==============================================================
+INFO  13:28:01 Successfully provisioned 4 aws-ec2 machines
+INFO  13:28:01 ==============================================================
+INFO  13:28:01 Shutting down Provisioner...
+INFO  13:28:01 Done!
+```
+Also you can see `agents.txt` was created automatically by provisioner in the working directory. This `agents.txt` file includes ips of the EC2 instances.
+
+* After the create instances and install agents to them. You need cordinate and configure your test suite via `cooordinator`.
+	
+	```
+	coordinator     --memberWorkerCount 2 \
+                --workerVmOptions "-ea -server -Xms2G -Xmx2G -verbosegc -XX:+PrintGCDetails -XX:+PrintGCTimeStamps -Xloggc:gc.log -XX:+HeapDumpOnOutOfMemoryError" \
+                --hzFile            hazelcast.xml \
+                --clientWorkerCount 2 \
+                --clientWorkerVmOptions "-ea -server -Xms2G -Xmx2G -verbosegc -XX:+PrintGCDetails -XX:+PrintGCTimeStamps -Xloggc:gc.log -XX:+HeapDumpOnOutOfMemoryError" \
```

Comment:
There does not exists a file 'client-hazelcast.xml' and 'hazelcast.xml' in the current director. Did you try this at all?